src/bun.js/ConsoleObject.zig

```diff
@@ -324,7 +324,7 @@ pub const TablePrinter = struct {
         var width: usize = 0;
         var value_formatter = this.value_formatter;
 
-        const tag = ConsoleObject.Formatter.Tag.get(value, this.globalObject);
+        const tag = ConsoleObject.Formatter.Tag.get(value, this.globalObject) catch return 0; // TODO
```

Comment:
looks like all getWidthForValue callsites use JSError so i think this can be changed to `try`